src/index.html

```diff
@@ -2,11 +2,414 @@
+
+            <li class="nav__element">
+              <a
+                href="#"
+                class="nav__element-link"
+              >
+              Samsung
+              </a>
+            </li>
+
+            <li class="nav__element">
+              <a
+                href="#"
+                class="nav__element-link"
+              >
+              Smartphones
+              </a>
+            </li>
+
+            <li class="nav__element">
+              <a
+                href="#"
+                class="nav__element-link"
+                data-qa="nav-hover"
+              >
+              Laptosps & Computers
+              </a>
+            </li>
+
+            <li class="nav__element">
+              <a
+                href="#"
+                class="nav__element-link"
+              >
+              Gadgets
+              </a>
+            </li>
+
+            <li class="nav__element">
+              <a
+                href="#"
+                class="nav__element-link"
+              >
+              Tablets
+              </a>
+            </li>
+            <li class="nav__element">
+              <a
+                href="#"
+                class="nav__element-link"
+              >
+              Photo
+              </a>
+            </li>
+            <li class="nav__element">
+              <a
+                href="#"
+                class="nav__element-link"
+              >
+              Video
+              </a>
+            </li>
+
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <div class="card-container">
+        <div
+        class="card"
+        data-qa="card"
+      >
+          <img
+          src="images/imac.jpeg"
+          alt="product item imac"
+          class="card__photo"
+        >
+          <h2 class="card__title">APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)</h2>
+          <span class="card__description">Product code: 195434</span>
+          <div class="card__review">
+
+            <ul class="card__review-stars stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+
+            <span class="card__review-raiting">Reviews: 5</span>
+          </div>
+
+          <div class="card__price">
+            <span class="card__price-value">Price:</span>
+            <span class="card__price-amount">$2,199</span>
+          </div>
+
+          <a
+          class="card__buttom"
+          data-qa="hover"
+        >
+          Buy
+          </a>
+
+        </div>
+        <div
+        class="card"
+        data-qa="card"
+        >
+          <img
+          src="images/imac.jpeg"
+          alt="product item imac"
+          class="card__photo"
+          >
+          <h2 class="card__title">APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)</h2>
```

Comment:
wrong formatting of `div` and `img`

below code has the same errors,  please fix them